CHANGELOG.md

```diff
@@ -2,6 +2,9 @@ Changelog
 =========
 
 * Upgrade to TChannel 1.2.0 and add Open Tracing support.
+* Add support for -H key:value header arguments.
```

Comment:
Suggest:

* Add support for headers with argument `-H key:value`.
* Add support for baggage with argument `-B key:value` when `--jaeger` is passed.